selfdrive/controls/controlsd.py

```diff
@@ -486,9 +486,9 @@ def controlsd_thread(sm=None, pm=None, can_sock=None):
 
   # Write CarParams for radard and boardd safety mode
   cp_bytes = CP.to_bytes()
-  params.put("CarParams", cp_bytes)
-  params.put("CarParamsCache", cp_bytes)
-  params.put("LongitudinalControl", "1" if CP.openpilotLongitudinalControl else "0")
+  put_nonblocking("CarParams", cp_bytes)
```

Comment:
Other services are waiting for carParams to start. Use synchronous version here.